cpp/arcticdb/async/tasks.hpp

```diff
@@ -395,26 +395,26 @@ struct SegmentFunctionTask : BaseTask {
 
 struct MemSegmentProcessingTask : BaseTask {
     std::vector<std::shared_ptr<Clause>> clauses_;
-    Composite<EntityIds> entity_ids_;
+    EntityIds entity_ids_;
 
     explicit MemSegmentProcessingTask(
            std::vector<std::shared_ptr<Clause>> clauses,
-           Composite<EntityIds>&& entity_ids) :
+           EntityIds&& entity_ids) :
```

Comment:
I'm never sure whether aliasing a std::vector is a good idea, I spent some time wondering if EntityIds was a class that did something super smart with multiple EntityIds.  Unless of course the plan is to swap it out for something like that in the future.

Reply:
Agreed, I added it in originally because otherwise you would have had `std::vector<Composite<std::vector<EntityId>>>`, but now I'm ripping `Composite` out it'll be readable without the alias